lib/doorkeeper/orm/active_record/stale_record_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module Orm
+    module ActiveRecord
+      class StaleRecordCleaner
```

Comment:
Just a small hint: maybe better to name it '`StaleRecordsCleaner`' (as we remove more than one record)? :)

Reply:
True :)

fixed